back/sssscs/src/main/java/com/ib/certificate/ICertificateRequestRepo.java

```diff
@@ -0,0 +1,21 @@
+package com.ib.certificate;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import com.ib.certificate.CertificateRequest.Status;
+import com.ib.user.User;
+
+@Repository
+public interface ICertificateRequestRepo extends JpaRepository<CertificateRequest, Long> {
+	public Optional<CertificateRequest> findByIdAndStatusEquals(Long id, Status status);
+	public List<CertificateRequest> findByIssuer(User issuer);
+	
+	@Query("select r from CertificateRequest r left join r.parent c where ((c.issuer.id = :issueeId) or (:includeEmpty = true and c = null))")
+	public List<CertificateRequest> findByIssuee(@Param("issueeId") Long issueeId, @Param("includeEmpty") boolean includeEmpty);
```

Comment:
`@Param` is redundant

Reply:
I had to manually toggle a checkbox which effectively tells Eclipse to pass `-parameters` to be able to work without `@Param`. Done in [e54fb47](https://github.com/magley/sssscs/pull/3/commits/e54fb477a8cacf0800c8eb8535a34ae71530493c).